server/webapp/WEB-INF/rails.new/app/controllers/analytics_controller.rb

```diff
@@ -24,14 +24,15 @@ class AnalyticsController < ApplicationController
 
   def index
     @view_title = 'Analytics'
-    @plugin_ids = default_plugin_info_finder.allPluginInfos(PluginConstants.ANALYTICS_EXTENSION).inject([]) do |memo, plugin|
-      memo << plugin.getDescriptor().id() if plugin.getCapabilities().supportsAnalyticsDashboard()
+    @plugin_ids = default_plugin_info_finder.allPluginInfos(PluginConstants.ANALYTICS_EXTENSION).inject({})do |memo, plugin|
```

Comment:
This should no longer be called `@plugin_ids`? Right, @kierarad and @maheshp? It looks like it's a hash.